.github/workflows/recommend-integration-tests.yml

```diff
@@ -23,7 +23,7 @@ jobs:
           if [ -z "$DIFF" ]; then
             echo "diff=" >> $GITHUB_OUTPUT
           else
-            echo "diff=$DIFF" >> $GITHUB_OUTPUT
+            echo "diff=true" >> $GITHUB_OUTPUT
```

Comment:
Can this be changed back or is this still causing an issue in this PR?

Reply:
Done!